recipes/chembl_webresource_client/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "chembl_webresource_client" %}
+{% set version = "0.9.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/5d/43/bf0a4d475be9b16f1cbecf0dfe5cb2652305ad239d726a9238d1c4d1713d/chembl-webresource-client-0.9.31.tar.gz
+  sha256: 7a51a158271f569c1b20354541ee10988f060bcdbe1ce9e76f0a0d76e57beb10
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - easydict
+    - grequests
+    - pip
+    - python >=3.5,<3.7
```

you can pin just ``3.6`` then
Why this project does not support Python 3.7?

you can do something like:
```suggestion
    - python 3.6.*
```

---

Thanks for the quick review.
There seem to be a couple of reasons, see https://github.com/chembl/chembl_webresource_client/issues/40